packages/web/components/Landing/WildWeb.tsx

```diff
@@ -58,18 +58,23 @@ export const WildWeb: React.FC = () => {
             fontSize={{ base: 'xl', '2xl': '3xl' }}
           >
             A new world is being built but it's{' '}
-            <Text as="strong">hard to navigate.</Text>
+            <Text as="strong" className="gradient-text">
+              hard to&nbsp;navigate.
+            </Text>
           </Text>
           <Text pb={{ base: '1.188rem', '2xl': '2.188rem' }}>
             The resources, building blocks, &amp; tools are all over the place{' '}
-            <Text as="strong">but the maps are inexistent</Text>.
+            <Text as="strong" className="gradient-text">
+              but the maps are inexistent
+            </Text>
+            .
           </Text>
           <Text pb={{ base: '1.188rem', '2xl': '2.188rem' }}>
-            There are pitfalls, gold rushing cowboys, &amp; snake oil salesmen
-            at every corner.
+            There are pitfalls, gold rushing cowboys &amp; snake oil salesmen at
```

Comment:
Why'd you remove the Oxford comma & one before it?

```suggestion
            There are pitfalls, gold, rushing cowboys, &amp; snake oil salesmen at
```

Reply:
I am actually an "I'll give you my Oxford comma from when you pry it from my cold, dead hands" type of person.

Removing it was in the task though, and I didn't think too much about that.

Thinking of it more, is it because it looked weird followed by the "&"?

Suggestion

> There are pitfalls, gold-rushing cowboys, and snake oil salesmen at

or even

> There are pitfalls, gold-rushing cowboys, and snake-oil salesmen at

And now I think about it, do cowboys rush for gold or do they look after their cows? I get the general sentiment, though.